recipe-client-addon/lib/NormandyApi.jsm

```diff
@@ -62,7 +62,10 @@ this.NormandyApi = {
   },
 
   async getApiUrl(name) {
-    const apiBase = prefs.getCharPref("api_url");
+    let apiBase = prefs.getCharPref("api_url");
+    if (!apiBase.endsWith('/')) {
+      apiBase += '/';
```

Comment:
This is fine if `apiBase` is guaranteed to never include query parameters or fragments. And if you can guarantee that, you can guarantee that it ends in a slash, no?